packages/btc/src/transaction/build.ts

```diff
@@ -237,6 +247,8 @@ export class TxBuilder {
         address: props.address,
         targetAmount: _targetAmount,
         allowInsufficient: true,
+        noUtxosCache: true,
+        cacheKey: props.cacheKey,
```

Comment:
It's a bit confusing to pass a cacheKey while `noUtxosCache=true`.

As my understanding to the implementation,
`collectSatoshi` uses cache by default, only the `cache getter()` does not use cache.

Reply:
Ok, I get it.

There are actually two layers of cache in the implementation:
- one is sdk's own cache
- The second layer is the assets-api cahce

Reply:
Then it might be better to rename `noUtxosCache` to `noAssetsApiCache`.

Reply:
Options has been renamed: https://github.com/ckb-cell/rgbpp-sdk/pull/184/commits/092e3166195a38c064145f14d46f3697919a6e8b